share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -725,7 +725,7 @@ tutorial-build:
 
 ml-linux-x86_64-cpu-generate:
   extends: [ ".generate-x86_64", .ml-linux-x86_64-cpu, ".tags-x86_64_v4" ]
-  image: ghcr.io/spack/linux-ubuntu22.04-x86_64_v2:v2024-01-29
+  image: ghcr.io/spack/ubuntu-22.04:main
```

Comment:
I think this should have a date tag on it right?

Reply:
We haven't created any date tags yet. Is there any reason not to just use main since we control the pipeline and the image?

Reply:
my main concerns are just tracking and avoiding accidental upgrades. One of the worst things to have to debug in CI is an untracked container update that makes a minor breaking change...If we can get a date tag that would be good.

Reply:
@haampie @alalazo does one of you have permission to create a new release/tag on https://github.com/spack/gitlab-runners?

Reply:
@adamjstewart https://github.com/spack/gitlab-runners/actions/runs/8988272315